assessment.md

```diff
@@ -143,39 +143,96 @@ TODO: add description for each assessment level for this skill.
+* Good = Programming style is fairly thorough and mostly uses proper naming conventions, indentations, and a fair amount of commenting. Styling is mostly consistent at this level.
+* Excellent = Programming style is very thorough and uses proper naming conventions, indentations, and is descriptively commented throughout. Styling is consistent at this level.
 
 * Managing software complexity
 
-TODO: add description for each assessment level for this skill.
+Managing software complexity will be evaluated on the following scale:
+
+* None = Written software is at a state where once the author has left the code, it is quite possible they would not be able to recognize what they left behind. Complexities reach a level so high that code becomes illegible to any reader.
+* Inadequate = Written software is not efficient in nature and often strays into unnecessary complexities; conditional statements are frequently overly convoluted and hard-coding is consistently present throughout.
+* Adequate = Written software is hardly efficient in nature and occasionally strays into unnecessary complexities; conditional statements can occasionally be overly convoluted and hard-coding is frequently present.
+* Good = Written software is decently efficient in nature and barely strays into unnecessary complexities; conditional statements are often not overly convoluted and hard-coding is only semi-present at best.
+* Excellent = Written software is incredibly efficient in nature and does not stray into unnecessary complexities or perhaps enter complexity without reason; conditional statements are not overly convoluted and hard-coding is hardly, if at all present.
 
 * Mitigating software risk
 
-TODO: add description for each assessment level for this skill.
+Ability to plan for and mitigate risks.
+
+* None = Does not attempt to mitigate or identify potential risks. Does not make an effort to communicate with teammates or handle problems.
+* Inadequate = Makes little attempt to mitigate or identify risks. Low effort in communication and planning.
+* Adequate = Makes an attempt to mitigate and identify risks. Communicates with teammates about some risks but may not be adequately prepared.
+* Good = Makes a good attempt at identifying and mitigating potential risks. Takes some early action to prevent problems. Often communicates with teammates about potential issues. Is often able to handle unexpected problems.  
+* Excellent = Identifies potential risks and actively works to avoid them. Communicates with group members to effectively prevent and manage any risks or problems. Is able to handle unexpected problems when they do appear and asks for help when appropriate.
 
 * Software testing
 
-TODO: add description for each assessment level for this skill.
+Understanding and successfully completing software testing through the correct
+installation of the necessary tools needed to test the software:
+
```

Comment:
I like this description but in order to maintain consistency with other groups, you should either remove them or ask all other groups to include a description after every skill. You can also incorporate some of these ideas in the assessment scales